Utils/github_workflow_scripts/autobump_rn.py

```diff
@@ -0,0 +1,1170 @@
+    ) -> ConditionResult:
+        """Checks if packs major changed.
+        Args:
+            previous_result: previous check artifacts.
+
+        Returns(ConditionResult): whether the condition check pass,
+            or we should skip this pr from auto-bumping its release notes, with the reason why to skip.
+        """
+        origin_pack_metadata_version = Version(
+            self.origin_base_metadata.get(
+                Metadata.CURRENT_VERSION, self.DEFAULT_VERSION
+            )
+        )
+        branch_pack_metadata_version = Version(
+            self.branch_metadata.get(Metadata.CURRENT_VERSION, self.DEFAULT_VERSION)
+        )
+        if origin_pack_metadata_version.major != branch_pack_metadata_version.major:
+            return ConditionResult(
+                should_skip=True,
+                reason=self.generate_skip_reason(
+                    origin_version=origin_pack_metadata_version,
+                    branch_version=branch_pack_metadata_version,
+                ),
+            )
+        else:
+            return ConditionResult(should_skip=False)
+
+
+class MaxVersionCondition(MetadataCondition):
+    MAX_ALLOWED_VERSION = "99"
+
+    def __init__(
+        self,
+        pack: str,
+        pr: PullRequest,
+        git_repo: Repo,
+        branch_metadata: Dict,
+        origin_base_metadata: Dict,
+    ):
+        super().__init__(
+            pack=pack,
+            pr=pr,
+            git_repo=git_repo,
+            branch_metadata=branch_metadata,
+            origin_base_metadata=origin_base_metadata,
+        )
+
+    def generate_skip_reason(       # type: ignore[override]
+        self, origin_version: str, branch_version: str, **kwargs
+    ) -> str:
+        """
+        Args:
+            origin_version: pack version in the base branch (origin/master).
+            branch_version: pack versio in the branch.
+        Returns: Reason why the condition failed, and pr skipped.
+        """
+        return SkipReason.EXCEED_MAX_ALLOWED_VERSION.format(
+            self.pack, self.MAX_ALLOWED_VERSION, origin_version, branch_version
+        )
+
+    def _check(
+        self, previous_result: Optional[ConditionResult] = None, **kwargs
+    ) -> ConditionResult:
+        """Checks if packs version is 99. (99 is the last supported version number).
+        Args:
+            previous_result: previous check artifacts.
+
+        Returns(ConditionResult): whether the condition check pass,
+            or we should skip this pr from auto-bumping its release notes, with the reason why to skip.
+        """
+        origin_pack_metadata_version = self.origin_base_metadata.get(
+            Metadata.CURRENT_VERSION, self.DEFAULT_VERSION
+        )
+        branch_pack_metadata_version = self.branch_metadata.get(
+            Metadata.CURRENT_VERSION, self.DEFAULT_VERSION
+        )
+        if (
+            self.MAX_ALLOWED_VERSION in origin_pack_metadata_version
```

Comment:
`branch_pack_metadata_version`'s default is `self.DEFAULT_VERSION`, does `in` work in this case?

Reply:
The default version is 1.0.0 and get. metadata.current version will return the pack version. Why the `in` functionality should be different?